src/shared/data/amendments_info.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Why is this empty?

Reply:
This is to manually add amendment info for amendments that are not on xrpl.org or the rippled files we use (probably ones that are on sidechain). I have this setup first as an empty array but we can add data later on when needed.

Reply:
This only needed if there's a need so I don't think we need a TODO